src/Voron/Debugging/StorageReportGenerator.cs

```diff
@@ -121,15 +125,37 @@ public DetailedStorageReport Generate(DetailedReportInput input)
             };
 
             var tempBuffers = GenerateTempBuffersReport(input.TempPath, input.JournalPath);
+            var preAllocatedBuffers = GetReport(new NewPageAllocator(_tx, _tx.RootObjects), input.IncludeDetails);
+
+            if (input.IncludeDetails == false && _skippedStreamsDetailsEntry != null)
+            {
+                // we don't have the actual trees' streams size at this point
+                // so we calculate the original size as if we read the streams by:
+                // [DataFile allocated space] - [DataFile free space] - [Tables allocated space] - [FixedTrees allocated space] - [pre allocated buffers space] 
+
+                var skippedStreamsAllocatedSpaceInBytes = dataFile.AllocatedSpaceInBytes - dataFile.FreeSpaceInBytes - _tablesAllocatedSpaceInBytes  - preAllocatedBuffers.OriginallyAllocatedSpaceInBytes - _fixedTreesAllocatedSpaceInBytes ;
+                _skippedStreamsDetailsEntry.AllocatedSpaceInBytes = skippedStreamsAllocatedSpaceInBytes;
+                _skippedStreamsDetailsEntry.Length = skippedStreamsAllocatedSpaceInBytes;
+
+                foreach (var tree in trees)
+                {
+                    if (tree.Streams?.Streams != null && tree.Streams.Streams[0].Name == "Stream details summary info")
```

Comment:
This ""Stream details summary info"" should go to a const and be used in both places

Reply:
done